modules/admin-ui-frontend/app/scripts/modules/registration/partials/terms-of-use.html

```diff
@@ -23,7 +23,40 @@
   agree to sending data from all categories.
 </p>
 
+<h1 style="font-size: x-large;margin: 15px 0">
+  Data Stewardship
+</h1>
+
+<p style="padding-bottom: 10px; font-weight: bold">
+  Location
+</p>
+<p style="padding-bottom: 15px">
+  This data will be stored within the European Union, and will be GDPR compliant.
+  The current location can be found <a href="https://docs.opencast.org/develop/developer/#infrastructure/">here</a>
+</p>
+
+<p style="padding-bottom: 10px; font-weight: bold">
+  Storage Duration
+</p>
+<p style="padding-bottom: 5px;">
+  Organizational Profile
+</p>
+<p style="padding-bottom: 10px">
+  Your organizational profile will be purged 6 months after its last update.
+</p>
+<p style="padding-bottom: 5px">
+  Usage statistics and technical data
+<p style="padding-bottom: 15px">
+  Your technical statistics will be purged 24 months after their last update
+</p>
 
+<p style="padding-bottom: 10px; font-weight: bold">
+  Access to Information
+</p>
+<p style="padding-bottom: 15px">
+  Acess to the raw data is restricted to the Opencast board, who can authorize release to commiters on a case-by-case basis.
```

Comment:
We could include a link to the Governance document explaining how the board is formed and who is currently on the board. See: https://docs.opencast.org/develop/developer/#governance/#current-board

Reply:
IMO this is out of scope for this PR. If someone wants to know about how the sausage is made, they'll find it.